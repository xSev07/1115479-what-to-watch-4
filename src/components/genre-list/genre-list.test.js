import React from "react";
import renderer from "react-test-renderer";
import GenreList from "./genre-list";

it(`Should GenreList render correctly`, () => {
  const tree = renderer
    .create(
        <GenreList
          genres={[`genre1`, `genre2`, `genre3`]}
          activeGenre={`genre2`}
          onClick={() => {}}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
