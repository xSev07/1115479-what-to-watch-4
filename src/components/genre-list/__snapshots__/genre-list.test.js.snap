// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should GenreList render correctly 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Genre1
    </a>
  </li>
  <li
    className="catalog__genres-item catalog__genres-item--active"
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Genre2
    </a>
  </li>
  <li
    className="catalog__genres-item "
  >
    <a
      className="catalog__genres-link"
      href="#"
      onClick={[Function]}
    >
      Genre3
    </a>
  </li>
</ul>
`;
